### PR TITLE
update container shorthand

### PR DESCRIPTION
**Pull request by meistudioli**
Make `container` shorthand clear to understand.

### CODE REVIEW

File: files/en-us/web/css/css_container_queries/index.md [hunk 1]
```diff
@@ -42,8 +42,8 @@ In addition the draft specification proposes some new properties:
   - : Defines an element as a **query container**. Descendants can query aspects of its sizing, layout, style and state.
 - `container-name`
   - : Specifies a list of **query container names** for `@container` rules to use to filter which query containers are targeted.
-- `container`:
-  - : A shorthand property to set both `container-type` and `container-name`.
+- `container`: `<'container-name'> [ / <'container-type'> ]?`
```
Comment (estelle) [inline files/en-us/web/css/css_container_queries/index.md:45]:
```suggestion
- `container`: 
```

File: files/en-us/web/css/css_container_queries/index.md [hunk 2]
```diff
@@ -42,8 +42,8 @@ In addition the draft specification proposes some new properties:
...
+  - : A shorthand property to set both `container-name` and `container-type` (use slash as separate sign).
```
Comment (estelle) [inline files/en-us/web/css/css_container_queries/index.md:46]:
```suggestion
  - : A shorthand property to set both `container-name` and `container-type`. The syntax is `<'container-name'> [ / <'container-type'> ]?`,  with a slash between values when both are present.
```

**Review by estelle — COMMENTED** [review]
thank you for the contribution. Made suggestions so it better follows how MDN does it.

**Comment by Josh-Cena** [comment]
Pinging @estelle

File: files/en-us/web/css/css_container_queries/index.md [hunk 3]
```diff
@@ -44,7 +44,7 @@ In addition the draft specification proposes some new properties:
 - `container-name`
   - : Specifies a list of **query container names** for `@container` rules to use to filter which query containers are targeted.
 - `container`:
-  - : A shorthand property to set both `container-type` and `container-name`.
+  - : A shorthand property to set both `container-name` and `container-type`. The syntax is `<'container-name'> [ / <'container-type'> ]?`,  with a slash between values when both are present.
```
Comment (Josh-Cena) [inline files/en-us/web/css/css_container_queries/index.md:47]:
```suggestion
  - : A shorthand property to set both `container-name` and `container-type`. The syntax is `<'container-name'> [ / <'container-type'> ]?`, with a slash between values when both are present.
```

**Comment by bsmth** [comment]
Relates to https://github.com/mdn/content/pull/22186

**Review by estelle — APPROVED** [review]

**Pull request merged**